### ISSUE TITLE
Attachment Text states "File from Node-RED is attached" needs to be configurable

### ISSUE THREAD

**Issue by converseKarl** [node-red/node-red-nodes#826]
Sending an attachment forces the label "File from Node-RED is attached".  Other people have raised this on the Node-Red forums. It's a show stopper using this library forcing to show such labels isn't always ideal.

1. Either want no attachment text so set to msg.attachmenttext="";
2. Something custom   set to say msg.attachmenttext  = "See latest attachment"
3. then Default to File from Node-RED is attached

**Comment by dceejay** [comment]
Did you try msg.description as per the info sidebar ?

**Comment by converseKarl** [comment]
> Did you try msg.description as per the info sidebar ?

yes mate I did , no success it puts the description in the email body but still has File from Node-RED is attached included.  Thanks kindly for your suggestion :-)

**Comment by dceejay** [comment]
So why is this a show stopper ?

**Comment by hardillb** [comment]
And have you tried the method I suggested?

**Comment by dceejay** [comment]
Indeed the correct method is as per @hardillb  - the other is just a convenience.

**Comment by converseKarl** [comment]
> So why is this a show stopper ?

i've looked at the code of the build and the NODE-RED attachment statement is hardcoded by default, secondly the proposed solution which I will try (and Ben thanks for your kind input on that - I will try this over the next few days),  I'm actually writing a complex transcoder from one AI model to another, generating multiple file assets. Putting them into a buffer as recommended for multiple file attachment and then add them into an array of attachments (with own payload) as I create them, then finally into a single zip file (using zip node) as the last task before creating email attachment and sending. All works great then I get the message FILE FROM NODE-RED is attached and tried all ways to remove it. Did not expect stuff to be hard coded in this library as its clearly there.

I will share more info later if I struggle to remove this from Ben's recommendation.  Many thanks for the kind responses and support. Very much appreciated folks!

**Comment by Saminarizona** [comment]
I did not like the tag line "File from Node-RED is attached" either.    I found in file C:\Users\USERNAME\.node-red\node_modules\node-red-node-email\locales\en-US\61-email.json   

on mine it was line 48 

 "default-message": "__description__\n\nFile from Node-RED is attached: __filename__"

**Comment by dceejay** [comment]
It is only hard-coded if you send a binary buffer payload and  no other options. If you declare the msg object as Ben describes above the plain text msg.payload will become your message instead.
